meson.build

```diff
@@ -152,6 +152,28 @@ if is_cross_build
 	)
 endif
 
+# Ensure that MSVC is switched to standards compliant mode
```

Comment:
Perhaps better to put this logic in `src/platforms/hosted/meson.build`? That way you can directly fix up `bmda_args` appropriately and use `cc`; It also puts the logic in with the rest of the Windows build logic.